omero/themes/globalomerotoc.html

```diff
@@ -1,5 +1,5 @@
 <h3><a href="{{ pathto(master_doc) }}">{{ _('OMERO') }}</a></h3>
 {{ toctree()}}
 <a href="http://downloads.openmicroscopy.org/latest/omero/">{{ _('Downloads') }}</a></br>
-<a href="//www.openmicroscopy.org/site/products/omero/feature-list">{{ _('Feature List') }}</a></br>
-<a href="//www.openmicroscopy.org/site/about/licensing-attribution">{{ _('Licensing') }}</a>
+<a href="https://www.openmicroscopy.org/omero/new/">{{ _('Features') }}</a></br>
```

Comment:
I can add the docs index here too if the consensus is to remove the versioning banner as Seb suggested on the card